__tests__/pages/edukasi.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { render, screen } from "@testing-library/react";
+import EducationPage, { getStaticProps } from "~/pages/edukasi";
+import educations from "~/lib/content/educations";
+
+jest.mock("~/lib/content/educations");
+jest.mock("next/router", () => require("next-router-mock"));
+
+describe("EducationPage", () => {
+  const { topics } = educations;
+
+  it("renders the title, description and breadcrumbs correctly", async () => {
+    render(<EducationPage educations={educations} />);
+
+    const title = screen.getByRole("heading", {
+      name: /Laman Edukasi Covid-19/i,
+    });
+    expect(title).toBeVisible();
+
+    const description = screen.getByText(
+      /Kumpulan informasi terbaru COVID-19, tips menjaga kesehatan selama pandemi, vaksinasi, hingga penanganan jika Anda terkonfirmasi positif COVID-19./i,
+    );
+    expect(description).toBeVisible();
+
+    const breadcrumbs = screen.getByRole("link", {
+      name: /Laman Edukasi Covid-19/i,
+    });
+    expect(breadcrumbs).toBeVisible();
+    expect(breadcrumbs).toHaveAttribute("href", "/edukasi");
+  });
+
+  it("renders the topics correctly", () => {
```

Comment:
Add assertion to check index 0 is open by default.

Reply:
<img width="347" alt="Screen Shot 2021-08-13 at 06 38 31" src="https://user-images.githubusercontent.com/45778229/129283231-e14c25e2-25e0-4c86-b009-6330736adf7f.png">

it's already tested on the component, should I keep add assertion?

Reply:
Then maybe align the title of test, not jut render correctly.

should render correctly and show index 0